server/devtool/projectrepository-modeshape/src/main/java/org/eclipse/vorto/devtool/projectrepository/modeshape/utils/ProjectRepositoryModeshapeConstants.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2017 Bosch Software Innovations GmbH and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * The Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * Contributors:
+ * Bosch Software Innovations GmbH - Please refer to git log
+ */
+
+package org.eclipse.vorto.devtool.projectrepository.modeshape.utils;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.vorto.devtool.projectrepository.utils.ProjectRepositoryConstants;
+
+public class ProjectRepositoryModeshapeConstants {
+
+	public static final String VORTO = "vorto";
```

Comment:
Constant classes are somewhat "ugly". Usually constants belong to a certain class. From what I can see, most of your defined constants are used by the Modeshape repository implementation. Why not defining them there ?

Reply:
The constants are being used in multiple files, Modeshape repository implementation and Resource query modeshape implementation. 
That is the main reason I moved them to a separate constant file. 

As for the code being "ugly", I can look up some better coding practices to keep the constants and make the code more readable